### ISSUE TITLE
Button / input lack the outlines for accessibility with keyboard

### ISSUE THREAD

**Issue by lukicenturi** [rotki/rotki#10618]
### Problem
Currently the input / buttons components lack keyboard accessibility due to missing outlines, focus states for some elements.
For example this menu that has multiple buttons:
<img width="287" height="158" alt="Image" src="https://github.com/user-attachments/assets/13233642-9630-4c1e-84c4-750fc29c38ef" />

### Solution
- Show proper outline when an input / button is focused
